### PR TITLE
sslh: add patch to work with libconfig 1.7.3

### PR DESCRIPTION
**Pull request by pfpulux**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/sslh/template [hunk 1]
```diff
@@ -10,6 +10,7 @@ license="GPL-2.0-or-later"
 homepage="https://www.rutschle.net/tech/sslh/README.html"
 distfiles="https://github.com/yrutschle/sslh/archive/v${version}.tar.gz"
 checksum=2e457e59592f8e523cade8d9302b0fdc87f8ea0322beb674dd7f067547a93ea9
+patch_args=-Np1
```
Comment (ericonr) [inline srcpkgs/sslh/template:13]: This is not necessary anymore.

**Review by ericonr — COMMENTED** [review]
If possible, can you link the commits you used on top of the patch?

File: srcpkgs/sslh/template (hunk 1 above)
Comment (ericonr) [reply srcpkgs/sslh/template:13]: (we default to Np1)

Comment (pfpulux) [reply srcpkgs/sslh/template:13]: This patch is a merge from the last release until the patch from upstream, but only the files handle the config.

**Pull request merged**